modules/bundles/get_started.ipynb

```diff
@@ -164,8 +164,6 @@
     "        \"$import os\",\n",
```

Comment:
<p><code></code><code></p><p>Now let's start to define the config file for a regular training task. MONAI bundles support both <code>JSON</code> and <code>YAML</code> format, here we use <code>JSON</code> as the example.</p><p>According to the predefined syntax of MONAI bundle, <code>$</code> indicates an expression to evaluate, <code>@</code> refers to another object in the config content. For more details about the syntax in bundle config, please check: <a href="https://docs.monai.io/en/latest/config_syntax.html" target="_blank">https://docs.monai.io/en/latest/config_syntax.html</a>.</p><p>Please note that a MONAI bundle doesn't require any hard-coded logic in the config, so users can define the config content in any structure.</p><p>For the first step, import <code>os</code> and <code>glob</code> to use in the expressions (start with <code>$</code>), then define input / output environments and enable <code>cudnn.benchmark</code> for better performance.</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Sure, updated in the latest commit.

Thanks.